workshops/templates/workshops/event_form.html

```diff
@@ -3,21 +3,44 @@
 {% load breadcrumbs %}
 {% block breadcrumbs %}
     {% breadcrumb_main_page %}
-    {% breadcrumb_index_all_objects model %}
-    {% if object %}
-        {% breadcrumb_object object %}
-        {% breadcrumb_edit_object object %}
-    {% else %}
-        {% breadcrumb_new_object model %}
-    {% endif %}
+    {% breadcrumb_active title %}
```

Comment:
yes, as I wasn't passing in model or object. I can, just for the sake of the breadcrumbs.